File: baekyeonsu/크기가 작은 부분문자열.js
function solution(t, p) {
  let answer = 0;

  for (let i = 0; i <= t.length - p.length; i++) {
    if (parseInt(t.slice(i, i + p.length), 10) <= parseInt(p, 10)) answer += 1;
  }

  return answer;
}

File: kimbyulyi/PGS/Level1/크기가작은부분문자열.java
class Solution {
    public int solution(String t, String p) {
        int answer = 0;
        for (int i = 0; i < t.length() - p.length() + 1; i++) {
            if (Long.parseLong(t.substring(i, i + p.length())) <= Long.parseLong(p)) {
                answer++;
            }
        }
        return answer;
    }
}